roles/repository/defaults/main.yml

```diff
@@ -6,5 +6,7 @@ repository_key_files_directory: ""
 repository_keys: []
 repository_key_ids: {}
 
-repository_apt_phased_updates: false
 repository_apt_acquire_forceipv4: false
+repository_apt_install_recommendeds: false
```

Comment:
`repository_apt_install_recommends` would be more in line with the apt option name